engine/src/main/java/graphql/nadel/engine/execution/ServiceExecutor.java

```diff
@@ -164,7 +166,7 @@ private ServiceExecutionParameters buildServiceExecutionParameters(ExecutionCont
                 .executionId(executionContext.getExecutionId())
                 .cacheControl(executionContext.getCacheControl())
                 .serviceContext(serviceContext)
-                .hydrationCall(isHydrationCall)
+                .executionHydrationDetails(hydrationDetails)
```

Comment:
This works for the old engine as well.  Not that I am super fussed but it was do-able so I did it